test/check-application

```diff
@@ -96,6 +100,19 @@ def run_e2e(verbose, image, browser, cpus, memory, sit):
         # Now actually run the tests
         selenium_address = "10.111.112.10" if selenium else ""
 
+        if "DEBUG_TEST" in os.environ:
+            try:
+                subprocess.call(["vncviewer", selenium_address+":5901", "&"])
+                subprocess.call(["vncviewer", selenium_address+":5902", "&"])
+            except OSError as e:
+                if e.errno == os.errno.ENOENT:
+                    print("Please install vncviewer by \
+                          'dnf install tigervnc'.")
```

Comment:
Please avoid the line break here (and below), otherwise you have a lot of whitespace in the resulting string. The lines are short enough anyway.